### PR TITLE
CNV-9540: RN for high-performance template Windows support

### PR DESCRIPTION
**Pull request by sabrinajess**
Re: [CNV-9540](https://issues.redhat.com/browse/CNV-9540)
Release note addition for 4.9

Preview: https://deploy-preview-36383--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-9-release-notes.html#virt-4-9-new

### CODE REVIEW

**Comment by dominikholler** [comment]
@ksimon1 can you please review this PR?

**Comment by kbidarkar** [comment]
Installed CNV 4.9 and was able to list the highperformance templates.
`]$ oc get templates -n openshift | grep -i windows.*-highperformance`

LGTM

File: virt/virt-4-9-release-notes.adoc [hunk 1]
```diff
@@ -45,6 +45,7 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 //CNV-12858 SVVP previous 8914
 
 //CNV-9540 High performance windows
+High-performance templates are now available for Windows servers.
```
Comment (sjhala-ccs) [inline virt/virt-4-9-release-notes.adoc:48]: FYI, the link in the PR description points to the incorrect Jira story. After looking at the epic associated with [CNV-9540](https://issues.redhat.com/browse/CNV-9540), it seems that the high-performance templates are available for both Windows 10 and the Windows Server operating systems.

I would suggest rewording your release note to something like this: "High-performance virtual machine templates are now available for supported Windows operating systems."

The supported guest operating systems are listed in a separate section in the release notes. You could link to that section. Please check with QE for accuracy.

**Review by sjhala-ccs — COMMENTED** [review]
A few suggested changes, thanks Sabrina!

File: virt/virt-4-9-release-notes.adoc (hunk 1 above)
Comment (sabrinajess) [reply virt/virt-4-9-release-notes.adoc:48]: Thanks, Shika! (link fixed)
@kbidarkar Please review the suggested revision for accuracy. 
 "High-performance virtual machine templates are now available for supported Windows operating systems."

Comment (kbidarkar) [reply virt/virt-4-9-release-notes.adoc:48]: Yes, wording in the above comment looks more accurate now, as the command `oc get templates -n openshift | grep -i windows.*-highperformance` lists both windows-10 and windows servers.
